### DISCUSSION TITLE
Icons are missing, can I get them back?

### DISCUSSION THREAD

**Discussion by dennidoubles** [cytech/Home-Assistant-wundergroundpws#190] [Q&A] [answered]
Hi,

In the former version there were icons in the weather predictions:

![image](https://github.com/cytech/Home-Assistant-wundergroundpws/assets/71798417/4c7dc9b5-68d5-4e7f-ad2c-edf2db756a2e)

Unfortunately they are not in the latest version. Is it possible to get them back, instead of the way too long (translated) descriptions?

![image](https://github.com/cytech/Home-Assistant-wundergroundpws/assets/71798417/1eef50c4-5dfe-451a-aaa7-0f307edde0d3)

**Comment by dennidoubles** [comment]
Anyone know how to solve this please?

**Comment by cytech** [comment, marked as answer]
not gonna happen...
its a whole other set of logic to match those weather underground supplied icons to the forecast entity.
and with this forecast entity deprecation looming in the future, I am sure it will become impossible..

**Comment by dennidoubles** [comment]
I understand, that's a real pity. No I have not so logical descriptions instead of an icon.
Well okay, thanks for letting me know then.

**Comment by cytech** [comment]
I guess you could edit the corresponding language file under the wupws_translations directory
and change the daypart: narrative  to an empty string.
that would shorten the description to just today, tonight, tomorrow, etc...
`"daypart": {
    "expired": "Today Expired",
    "narrative": "",
    "precipChance": "Precipitation Probability",
    "qpf": "Precipitation Amount",
    "temperature": "Forecast Temperature",
    "windSpeed": "Average Wind"`

**Comment by mnneely** [comment]
Old discussion, but I used a Mushroom template card and set the icon to match the state of the weather entity with this code.

```

{{ 
{"clear-night": "mdi:weather-night", 
"cloudy": "mdi:weather-cloudy",
"fog": "mdi:weather-fog",
"hail": "mdi:weather-hail",
"lighting":  "mdi:weather-lightning",
"lightning-rainy":  "mdi:weather-lightning-rainy",
"partlycloudy": "mdi:weather-partly-cloudy",
"pouring": "mdi:weather-pouring",
"rainy": "mdi:weather-rainy",
"snowy": "mdi:weather-snowy",
"snowy-rainy": "mdi:weather-snowy-rainy",
"sunny": "mdi:weather-sunny",
"windy": "mdi:weather-windy",
"windy-variant": "mdi:weather-windy-variant",
"exceptional": "mdi:weather-cloudy-alert"
}
[states('weather.<STATION_ID>')] | default("mdi:cloud-question-outline") }}
```

Of course, you have to replace <STATION_ID> with your own value. By changing the dictionary values, you can also customize icon color in the Mushroom template card based on the state of your weather entry, e.g.

`”fog”: “grey”`.
